models/demos/yolov4/ttnn/downsample4.py

```diff
@@ -15,7 +15,13 @@ def __init__(self, model) -> None:
             torch_model = model.torch_model
         self.torch_model = torch_model
         self.conv1 = Conv(
-            torch_model, "down4.conv1", [1, 40, 40, 256], (2, 2, 1, 1), reshard=True, height_sharding=False
+            torch_model,
+            "down4.conv1",
+            [1, 40, 40, 256],
+            (2, 2, 1, 1),
+            reshard=True,
+            height_sharding=False,
+            deallocate=False,
```

Comment:
why deallocate=false is needed here?

Reply:
downblock 4 and down block 5 are deallocating inputs d3 and d4 but they are re-used in "neck" part of the model.
Deallocation now actually works so it gets deallocated and model fails